src/HCSR04.cpp

```diff
@@ -7,9 +7,11 @@
 #include "HCSR04.h"
 
 UltraSonicDistanceSensor::UltraSonicDistanceSensor(
-        int triggerPin, int echoPin) {
+        int triggerPin, int echoPin, int maxDistanceCm, uint32_t timeoutMicroSec) {
```

Comment:
Can we rename timeoutMicroSec to maxTimeoutMicroSec? That way it is clear that it might be shorter (due to maxDistanceCm).